appConfig.js

```diff
@@ -49,14 +49,14 @@ const config = {
         header: 'Best Books for Teens',
         description: 'Explore our annual selection of outstanding young adult titles.',
         heroImageUrl: 'http://staff-picks-dev.us-east-1.elasticbeanstalk.com/books-music-dvds/' +
-          'recommendations/staff-picks/src/client/images/desktop.banner.YA.FIN.png',
+          'recommendations/staff-picks/src/client/images/desktop.teens.cover.2017.png',
```

Comment:
Having or not having `staff-picks` doesn't change anything. We should pick one over the other though.

Reply:
in the URL or more generally?

Reply:
Yes, in the URL. I noticed in the image reference below, `staff-picks` isn't there, but both images load just fine.